openfeature-provider/js/README.md

```diff
@@ -78,6 +78,38 @@ The provider periodically:
 
 ---
 
+## Sticky Assignments
```

Comment:
I think the docs are good, but could do a little more to explain that currently it comes with a drastic latency penalty? Like a styled info box.